Exiled.API/Features/Player.cs

```diff
@@ -1816,11 +1816,11 @@ public void AddItem(Item item, int amount, IEnumerable<AttachmentIdentifier> ide
         /// <param name="items">The list of items to be added.</param>
         public void AddItem(IEnumerable<Item> items)
         {
-            IEnumerable<Item> enumerable = items.ToList();
-            if (enumerable.Any())
+            List<Item> enumerable = items.ToList();
```

Comment:
why still use ToList()? it not better

Reply:
The only real efficient change that could have been made to the original is saving enumerable.Count() as a local variable, and using that instead of the .Any() and .Count() so that it doesn't enumerate twice.

Reply:
Because i dont wanna use LINQ `.Count()`, and `List` enumerate is a bit efficient.
And i dont understand this `if` why do this check need?

Reply:
If i remove this `if`, and `list.Count` usage, so `.ToList()` become unnecessary, yes

Reply:
Sooooooo should I delete it?